stac_fastapi/pgstac/models/links.py

```diff
@@ -65,9 +65,7 @@ def url(self):
         # We need to remove the root path prefix from the path before
         # joining the base_url and path to get the full url to avoid
         # having root_path twice in the url
-        if (
-            root_path := self.request.scope.get("root_path")
-        ) and not self.request.app.root_path:
+        if root_path := self.request.scope.get("root_path"):
```

Comment:
I guess we can drop the comments about `self.request.app.root_path` if we are not using it any more

Reply:
@vincentsarago look good now? https://github.com/stac-utils/stac-fastapi-pgstac/pull/270/commits/f08a6ffafd2f24c8cb546a8296b00b18f6674910